libs/env/src/env.ts

```diff
@@ -537,8 +537,23 @@ const envDefinitions = {
     desc: "if defined, avoids bypass of the currentDevice in the store.",
   },
   NFT_CURRENCIES: {
-    def: "ethereum,polygon",
-    parser: stringParser,
+    def: [
+      "arbitrum",
+      "avalanche_c_chain",
+      "base",
+      "blast",
+      "bsc",
+      "celo",
+      "ethereum",
+      "fantom",
+      "linea",
+      "moonbeam",
+      "optimism",
+      "polygon",
+      "scroll",
+      "sei_network",
```

Comment:
Activating NFTs on those chains might make the synchronization process ~2 times longer on explorers with rate limits. 
Please make sure it's considered acceptable by product that rate limited chains go from 15 to 25 seconds per sync (including discovery at the "addAccount" level).
Also I'm not sure we have anything in the coin-integration of cosmos for the support of NFTs, so `sei_network` might not work unless the whole portfolio is built frontend wise.

Reply:
Yep totally agree, I'll talk to Product :) 

thanks 🙏🏻